core/templates/dev/head/pages/topics_and_skills_dashboard/TopicsListDirective.js

```diff
@@ -21,26 +21,45 @@ oppia.directive('topicsList', [
       restrict: 'E',
       scope: {
         getTopicSummaries: '&topicSummaries',
-        canDeleteTopic: '&userCanDeleteTopic'
+        canDeleteTopic: '&userCanDeleteTopic',
+        isInModal: '&inModal',
+        selectedTopicList: '='
```

Comment:
Name might need a bit of changing/documentation. Not sure what this means.

(I can understand what a selected topic is. But what is a selected topic list? Is it a (selected topic) list or a selected (topic list)? If the former perhaps just selectedTopics?)

Reply:
Its a list of objects containing ids and versions of topics selected by the user. Then, shall I rename it to selectedTopicsIdAndVersionList, or something like that?

Reply:
Done, renamed to selectedTopicsIdAndVersionList.